From d13e915954aed2593bc14d35f008e578b012cf93 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Tue, 13 Nov 2018 17:37:11 +0800
Subject: [PATCH] ls1046ardb:enable distro feature for eMMC configuration

Enable distro feature for eMMC configuration on ls1046ardb board.

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 configs/ls1046ardb_emmc_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/ls1046ardb_emmc_defconfig b/configs/ls1046ardb_emmc_defconfig
index 03aec708ba..2aeca61ad3 100644
--- a/configs/ls1046ardb_emmc_defconfig
+++ b/configs/ls1046ardb_emmc_defconfig
@@ -40,3 +40,4 @@ CONFIG_DM_SPI=y
 CONFIG_FSL_QSPI=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
+CONFIG_DISTRO_DEFAULTS=y
-- 
2.14.4

